index.html

```diff
@@ -1,313 +1,351 @@
-                <li>javascript</li>
-                <li>ruby</li>
                 <li>html</li>
-                <li>css</li>
-              </ul>
-              <div class="frameworkheading">
-                <img src="images/Rectangle 56.png" alt="" />
-                <h1>frameworks</h1>
-              </div>
-              <ul class="frameworkSkill">
-                <li>Bootstrap</li>
-                <li>Ruby on rails</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
-                <li>RSPec</li>
-              </ul>
-              <div class="skillotherheading">
-                <img src="images/circle.png" alt="" />
-                <h1>Skills</h1>
-              </div>
-              <ul class="skilOther">
-                <li>Codekit</li>
-                <li>github</li>
-                <li>Codepen</li>
-                <li>gitlab</li>
-                <li>Terminal</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
               </ul>
-          </section>
+            </div>
+            <button class="button" id="w6button">See Project</button>
+          </div>
+        </div>
+      </section>
+      <section class="about" id="about">
+        <div class="aboutme">
+          <h1>About me</h1>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <button class="button">Get My Resume</button>
+          <hr />
+        </div>
+      </section>
+      <section class="myskills">
+        <div class="myskillslanguage">
+          <div class="myskillHeadingImg">
+            <img src="SVG/kite.svg" alt="" />
+            <h1 class="myskillHeading">Language</h1>
+          </div>
+          <ul class="skill-List">
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="myskillslanguage">
+          <div class="myskillHeadingImg" id="framework">
+            <img src="SVG/Rectangle.svg" alt="" />
+            <h1 class="myskillHeading">Frameworks</h1>
+          </div>
+          <ul class="skill-List">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="myskillslanguage">
+          <div class="myskillHeadingImg">
+            <img src="SVG/circle.svg" alt="" />
+            <h1 class="myskillHeading">Skills</h1>
+          </div>
+          <ul class="skill-List">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+            <li>Codepen</li>
+          </ul>
+        </div>
```

Comment:
- [ ] To match Figma design kindly fix these issues : 

1. Kindly remove the dots before each technologies item 
2. kindly make sure that the technologies lists are aligned to the left 


Figma : 

![image](https://user-images.githubusercontent.com/89282221/194702317-03e245ff-b45e-4788-a1d5-d48439766ef1.png)

Your Design:

![image](https://user-images.githubusercontent.com/89282221/194702296-bc692eb0-fe7f-4708-a014-71a63b7b2946.png)